### PR TITLE
fix context menus in ie10 not accessible

### PR DESCRIPTION
**Pull request by 0xF013**
[`* `#16028 ` context menus in ie10 not accessible`](http://www.openproject.org/work_packages/16028)

I am not sure how we should go on with this. IE10 allows only for specific elements to be focused (inputs, anchors), and not divs. I'll look into the focusing divs, not sure at the moment. This is the reason I didn't touch the right menu (the one with queries), as it focuses on divs and if I change it to anchors, it'll look uglier.

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage. <a href='http://0xf013-feature-16028-context-menus-in-ie10-not-accessible.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Click to open.</a>

File: app/assets/javascripts/lib/jquery.trap.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ IS" AND ANY EXPRESS OR IMPLIED WARRANTIES ARE DISCLAIMED.
 */
 
 /*jshint boss: true, bitwise: true, curly: true, expr: true, newcap: true, noarg: true, nonew: true, latedef: true, regexdash: true */
-    
+
```
Comment (hschink) [inline app/assets/javascripts/lib/jquery.trap.js:26]: Please put all code layout specific changes in a separate commit.

**Comment by hschink** [comment]
@0xF013 specs fail

**Comment by hschink** [comment]
Closed in favor for #2001

**Pull request closed without merging**